### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by pponce** [nicoduj/homebridge-harmony#125]
**Describe the bug**
Not sure if this is a bug or not. I’m running v.70  and using TvMode.
When I select an input other than the input that I set as the main input to be controlled with the “main” on off switch in the home app the tv switch does not show as having been turned on in the home app. The activity gets executed so everything works but the status of the TV in the home app remains as off.
Then when using a scene to turn off my tv and turn off the harmony activity it won’t turn it off because HomeKit thinks it was off all along.

**To Reproduce
Open home app and select one of the non main activities that appear as inputs. 

**Expected behavior**
When selecting an non main activity as input I expect the TV switch in the home app to show as “ON”

**IOS (please complete the following information):**
- Version 12.2
- Device:  iPhone XS

**Additional context**
Perhaps there is a config option I’m missing to enable what I think is expected behavior.

**Comment by nicoduj** [comment]
Hi, thank you for reporting. Will check but seems like a bug to me indeed . 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pponce** [comment]
I’m going to test it a bit more. It might just be a long delay that I’m seeing because later in the day I tried again and saw the same issue but after watching for about 1 hour i noticed it did finally engage and show as being on. Not sure how long it took.
I’ll see if it keeps happening and see how long the delay is and if perhaps re-opening the home app impacts things.

**Comment by nicoduj** [comment]
HomeKit does some update by its side sometimes (and while reopening / changing rooms also but not always) , so it might have set it after some time to on since an active input was set . I probably just need to update silently the on state while activating an activity (except the off activity) . 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pponce** [comment]
Yeah. I’m familiar with what you are describing in terms of when the home app updates and how to trigger an update by changing rooms.
I was definitely patient when and tried changing rooms when I saw that the state was not changing to “on”.  The previous night I ran into an issue where my scene didn’t turn off the tv so when I noticed today that the on state didn’t take I thought this was why the scene failed.  But then I saw that it did end up changing one to on at least one time for me. So it might be time related. I’ll play around some more and report back tomorrow
